superset/dashboards/permalink/commands/create.py

```diff
@@ -40,21 +53,28 @@ def __init__(
         self.dashboard_id = dashboard_id
         self.state = state
 
+    @property
+    def value(self) -> DashboardPermalinkValue:
+        return {
+            "dashboardId": self.dashboard_id,
+            "state": self.state,
+        }
+
+    @property
+    def signature(self) -> Tuple[Optional[int], DashboardPermalinkValue]:
```

Comment:
Why is the signature a function of the user?

Reply:
Just thought it'd be safer to give each user a unique permalink in case we want to anything special about it. (Can't think of a use case just yet) Plus user id is also saved for KeyValue so you can also argue this is for completeness sake.